### ISSUE TITLE
redis doesn't read in the last db dump

### ISSUE THREAD

**Issue by spratt** [SteamedPears/Code-Review#115]
I've heard that redis over-writes the dump immediately upon starting up.  We should fix this.

**Comment by psimonyi** [comment]
I'm guessing that the problem is just that the stop script kills redis so it doesn't write the dump on shutdown.

**Comment by spratt** [comment]
Nope!  I can see the data in the dump file...redis just doesn't read in the data from the dump when it starts up.

**Comment by psimonyi** [comment]
It's the case for me. Make 10 database transactions, wait 5 minutes, and restart the server. Redis loads the dump just fine. But if I don't wait, redis is killed before it writes the data to the file, and the entries get lost.

**Comment by psimonyi** [comment]
@spratt: Is this fixed now?
